RealifeTech-SDK.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,34 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		031A942E26613B2E00F12604 /* sell_fragment_product.graphql.swift in Sources */ = {isa = PBXBuildFile; fileRef = 031A941226613B2D00F12604 /* sell_fragment_product.graphql.swift */; };
```

Comment:
The version number needs to be incremented

Reply:
done